teach/templates/clubs/email/create_club_user/body.txt

```diff
@@ -0,0 +1,9 @@
+{{ username }},
```

Comment:
should this not be `{{ full_name }}`, since we're addressing a real person here, not just a webmaker user?